internal/mining/mining.go

```diff
@@ -1039,6 +1113,14 @@ func NewBlkTmplGenerator(policy *Policy, txSource TxSource,
 	}
 }
 
+// calcFeePerKb returns an adjusted fee per kilobyte taking the provided
+// transaction and its ancestors into account.
+func calcFeePerKb(txDesc *TxDesc, ancestorStats *TxAncestorStats) float64 {
+	txSize := uint32(txDesc.Tx.MsgTx().SerializeSize())
```

Comment:
No need to cast this to a `uint32` since you are just casing it to a `int64` below anyway

Reply:
fixed